llumnix/llumlet/llumlet.py

```diff
@@ -226,22 +227,40 @@ async def abort(self, request_id: Union[RequestIDType, Iterable[RequestIDType]])
     async def migrate_out(
         self,
         dst_instance_actor: ray.actor.ActorHandle,
-        dst_instance_id: str,
-        migration_type: Optional[MigrationType] = None
-    ) -> List[RequestIDType]:
-        migrated_request_ids = await self.migration_coordinator.migrate_out(
-            dst_instance_actor, dst_instance_id, migration_type)
-
-        if migration_type == MigrationType.PRE_STOP_MIGRATION:
-            if self.unit_status != UnitStatus.MIGRATING:
-                self.set_unit_status(UnitStatus.MIGRATING)
-            instance_info: InstanceInfo = self.backend_engine.get_instance_info()
-            num_running_requests = instance_info.num_running_requests
-            num_waiting_requests = instance_info.num_waiting_requests
-            if num_running_requests == 0 and num_waiting_requests == 0:
-                self.set_unit_status(UnitStatus.STOPPED)
-
-        return migrated_request_ids
+        dst_instance_context: InstanceContext,
+        migration_type: Optional[MigrationType] = None,
+        blocking: bool = True
+    ) -> None:
+        async def _inner_migrate_out(
```

Comment:
move _inner_migrate_out out as _migrate_out, and test _migrate_out rather than migrate out?